controllers/volumereplication_controller.go

```diff
@@ -109,6 +118,7 @@ func (r *VolumeReplicationReconciler) Reconcile(ctx context.Context, req ctrl.Re
 	if secretName != "" && secretNamespace != "" {
 		secret, err = r.getSecret(secretName, secretNamespace)
 		if err != nil {
+			_ = r.updateStatusStateAndMessage(instance, replicationv1alpha1.ReplicationFailure, err.Error())
```

Comment:
why are we ignoring this error?  (and if we are, then we can remove the `_` )

Reply:
I am ignoring the error here because we are returning other err anyway. The other error is much more important than this one.